macros/sql/get_tables_by_prefix.sql

```diff
@@ -1,13 +1,12 @@
 {% macro get_tables_by_prefix(schema, prefix, exclude='') %}
+  {{ return(adapter_macro('dbt_utils.get_tables_by_prefix', schema, prefix, exclude)) }}
```

Comment:
i don't think we need to use the `adapter_macro` here because there is only a single implementation of `get_tables_by_prefix` for all adapters.

Reply:
👍